.github/workflows/commitlint.yml

```diff
@@ -1,11 +0,0 @@
-name: Lint Commit Messages
```

Comment:
What is this being replaced with? Others that don't usually use conventional commits end up using it because this fails for them.

Reply:
Specifically, what is the github settings?

Reply:
[GitHub rulesets](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets).

It does not function exactly the same. The user only sees the message when clicking the merge button on their PR.
The error message is customizable. It currently says to use conventional commits and provides a link to the conventional commits website.